ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+### Document Issue
+
+**Document Location**: [PATH_TO_DOCUMENT](FULL_LINK)
+
+[//]: # (DELETE THIS LINE: If the document exists, replace PATH_TO_DOCUMENT with the path to the document header for example, path/name.md#header and replace FULL_LINK with full link of the path to the document header. If you are proposing a new document, replace PATH_TO_DOCUMENT with the path to the document header of where it should go.)
```

Comment:
Can you replace the current example with a link to a real document header within Community? Replace path/name.md... with a real one.